<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObjectDatabaseAdmin</class>
 <widget class="QMainWindow" name="GameObjectDatabaseAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Object Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <layout class="QHBoxLayout" name="toolLayout">
        <item>
         <widget class="QToolButton" name="logoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="toolExpander">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="headerLeftFixedSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="titleLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;h2&gt;Game Object Database&lt;/h2&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="headerRightFixedSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="welcomeLayout">
        <item>
         <spacer name="welcomeExpander">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="welcomeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Welcome, Administrator</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="recordLayout">
        <item>
         <widget class="QLabel" name="obectLabel">
          <property name="text">
           <string>Objects</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="commandLayout">
        <item>
         <widget class="QLabel" name="commandLabel">
          <property name="text">
           <string>Commands</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="commandBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Add Object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="generateButton">
             <property name="text">
              <string>Generate Objects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifyButton">
             <property name="text">
              <string>Modify Object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeButton">
             <property name="text">
              <string>Remove Object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeAllButton">
             <property name="text">
              <string>Remove All Objects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="viewButton">
             <property name="text">
              <string>View Object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="viewTablesButton">
             <property name="text">
              <string>View Table Objects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="errLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewTablesButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>viewTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeAllButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>removeAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logoutButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generateButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>generate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>GameObjectDatabaseAdmin</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>view()</slot>
  <slot>viewTable()</slot>
  <slot>viewAll()</slot>
  <slot>modify()</slot>
  <slot>removeAll()</slot>
  <slot>logout()</slot>
  <slot>remove()</slot>
  <slot>add()</slot>
  <slot>generate()</slot>
 </slots>
</ui>
